packages/inngest/src/components/InngestStepTools.ts

```diff
@@ -473,7 +479,21 @@ type InvocationTargetOpts<TFunction extends InvokeTargetFunctionDefinition> = {
 };
 
 type InvocationOpts<TFunction extends InvokeTargetFunctionDefinition> =
-  InvocationTargetOpts<TFunction> & TriggerEventFromFunction<TFunction>;
+  InvocationTargetOpts<TFunction> &
+    TriggerEventFromFunction<TFunction> & {
+      /**
+       * The step function will wait for the invocation to finish for a maximum
+       * of this time, at which point the retured promise will be rejected
+       * instead of resolved with the output of the invoked function.
+       *
+       * The time to wait can be specified using a `number` of milliseconds, an
+       * `ms`-compatible time string like `"1 hour"`, `"30 mins"`, or `"2.5d"`,
+       * or a `Date` object.
+       *
+       * {@link https://npm.im/ms}
+       */
```

Comment:
We may want to add a disclaimer to this like "The invoked function will not be cancelled after the timeout" - I could see some users thinking that this is tells _the invoked function to timeout_, so adding that could be clarifying. We definitely will need this within our docs when this is released.